dist/src/main/resources/application-rdbmsH2.yaml

```diff
@@ -84,6 +84,9 @@ zeebe:
     exporters:
       rdbms:
         className: io.camunda.exporter.rdbms.RdbmsExporter
+        args:
+          flushInterval: 5000
```

Comment:
I think above all for h2 we should use a lower flushInterval. Since this will mainly be used for local development. (500?)